### PR TITLE
Layout and PageBlock can now stretch and Box can use the container's full height

### PR DESCRIPTION
**Pull request by estacioneto**
#### What is the purpose of this pull request?

Allow `Layout` Component to use the container's full height when a `stretch` prop is passed.

#### What problem is this solving?

There was no way to make a `Layout` Component fill a container.

#### Screenshots or example usage

Layout Stretch

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- **Layout** can stretch using the `stretch` prop.
+- **PageBlock** can stretch using the `stretch` prop.
+- **Box** can have the container's full height using the `fullHeight` prop.
```
Comment (kevinch) [inline CHANGELOG.md]: Design review in progress.

**Comment by cmdalbem** [comment]
@estacioneto from what I recall from our discussion in Slack I think we can close this PR, right? :)

**Comment by cmdalbem** [comment]
@estacioneto up

**Comment by estacioneto** [comment]
Sorry about the delay @cmdalbem . Yes. It can be closed. If we need something similar later, we can reopen it :)

**Comment by cmdalbem** [comment]
@estacioneto no problems man! thanks for bringing the discussion, it was very enriching to everyone.

**Pull request closed without merging**